### ISSUE TITLE
Criterion 0.4.x trait incompatibility

### ISSUE THREAD

**Issue by cameronelliott** [jbreitbart/criterion-perf-events#22]
This is an awesome crate. I love being able to see instruction counts/cpu cycles etc for benches.

It looks like some changes are needed for Criterion 0.4.x

```
error[E0277]: the trait bound `Perf: Measurement` is not satisfied
   --> benches/my_bench.rs:31:18
    |
31  | fn bench(c: &mut Criterion<Perf>) {
    |                  ^^^^^^^^^^^^^^^ the trait `Measurement` is not implemented for `Perf`
    |
    = help: the trait `Measurement` is implemented for `WallTime`
note: required by a bound in `Criterion`
   --> /home/c/.cargo/registry/src/github.com-1ecc6299db9ec823/criterion-0.4.0/src/lib.rs:340:25
    |
340 | pub struct Criterion<M: Measurement = WallTime> {
    |                         ^^^^^^^^^^^ required by this bound in `Criterion`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `crit-perf` due to previous error
warning: build failed, waiting for other jobs to finish...
Error: 'cargo bench' returned an error (exit status: 101); unable to continue.
```

**Comment by jbreitbart** [comment]
Thanks for reporting the issue. :) Yeah looks like I need to update the crate.

**Comment by jbreitbart** [comment]
@cameronelliott Sorry for the delay. I have just released 0.3.0. This should work fine with the latest criterion release. Let me know if you have any further issues.
